addons/captives/functions/fnc_canBlindfoldCaptive.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: mrschick
+ * Tests if caller has a Contact DLC blindfold in their inventory to put on the captive
+ *
+ * Arguments:
+ * 0: caller (player) <OBJECT>
+ * 1: target <OBJECT>
+ *
+ * Return Value:
+ * The return value <BOOL>
+ *
+ * Example:
+ * [player, bob] call ACE_captives_fnc_canBlindfoldCaptive
+ *
+ * Public: No
+ */
+
+params ["_unit", "_target"];
+//Alive, handcuffed, not being escorted, and caller has a Contact DLC blindfold in their inventory
+
+(_target getVariable [QGVAR(isHandcuffed), false]) &&
+{isNull (attachedTo _target)} &&
+{alive _target} &&
+{(vehicle _unit) == _unit} &&
+{(vehicle _target) == _target} &&
+{(["G_Blindfold_01_black_F", "G_Blindfold_01_white_F"] findAny (items _unit)) != -1}
```

Comment:
probably make this config based?
like we've been doing with everything else
I can push this change

Reply:
Couldn't find the way ACE parses a CBA Setting String to Array, but I checked how [ACRE Animations](https://steamcommunity.com/sharedfiles/filedetails/?id=2140308792) does it and used their method of calling `setting = call compile setting`.